packages/devtools-source-map/package.json

```diff
@@ -8,12 +8,19 @@
   "bugs": "https://github.com/devtools-html/devtools-core/issues",
   "homepage": "https://github.com/devtools-html/devtools-core/tree/master/packages/devtools-source-map#readme",
   "main": "src/index.js",
+  "scripts": {
+    "copy-assets": "node bin/copy-assets"
+  },
   "dependencies": {
     "devtools-client-adapters": "^0.0.21",
     "devtools-config": "^0.0.12",
     "devtools-network-request": "^0.0.14",
     "md5": "^2.2.1",
     "source-map": "^0.5.6",
     "url": "^0.11.0"
+  },
+  "devDependencies": {
+    "devtools-launchpad": "^0.0.56",
+    "webpack": "1.14.0"
```

Comment:
Since this package's bundling scripts use `webpack`, it seemed best to explicitly state a dev dependency, like we do in debugger, launchpad, etc.